.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "semi": true,
+  "trailingComma": "none",
+  "singleQuote": true,
+  "printWidth": 80,
```

Comment:
Temat bardziej do dyskusji zespołowej ale jeśli wszyscy mamy duże ekrany proponowałbym co najmniej 120

Reply:
Agreed that 120 char line limit is better. Changed.